### PR TITLE
Update angular project page

### PR DESCRIPTION
**Pull request by paulbert**
Adding some more detail to the Angular page and updating the installation information.

rawgit:
http://rawgit.com/paulbert/paulbert.github.io/angular_project_2/#!pages/angular.md

### CODE REVIEW

File: pages/angular.md [hunk 1]
```diff
@@ -9,35 +9,24 @@ Create a prototype Progressive Web App using Angular & CouchDB with the BeLL App
...
-    curl -X PUT http://127.0.0.1:5984/_users
-    curl -X PUT http://127.0.0.1:5984/_replicator
-    curl -X PUT http://127.0.0.1:5984/_global_changes
-
-    ```
-
-4. Then go into the CouchDB dashboard at `http://127.0.0.1:5984/_utils` and enable CORS by clicking __Configuration__ -> __CORS__, click the __Enable CORS__ button and add `http://localhost:3000` and/or `http://127.0.0.1:3000` to the list
- 
-    *Paul's note: add image here*
+1. You should have Vagrant installed already, but if not install Vagrant
```
Comment (lmmrssa) [inline pages/angular.md]: could we also add the link for vagrant.
And yes virtual box as well something similar to BeLL-Apps documentation.
And one issue that happens with virtual box is that it needs to run on VT-x not just parallelism

**Comment by paulbert** [comment]
I found the link to the Vagrant install instructions and added it in.  Does that add enough?

**Comment by paulbert** [comment]
@LKhadka @xyb994 @alexkarasik @daisywatson @lucy811 Did you all get a chance to glance through this?  I'll merge this tomorrow unless one of you think I should make further adjustments.

**Comment by LKhadka** [comment]
Google has some free courses on PWAs. You might wanna include that https://codelabs.developers.google.com/codelabs/your-first-pwapp/#0
https://www.udacity.com/course/intro-to-progressive-web-apps--ud811

**Comment by xyb994** [comment]
https://rawgit.com/paulbert/paulbert.github.io/angular_project_2/#!./pages/angular.md#Working_on_a_component
> Here we'll focus on some key points you be aware of.

missing 'should'?

**Comment by paulbert** [comment]
Thanks for the feedback, everyone!

**Pull request merged**